tests/x11/libqt5_qtbase.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Tests for libqt5-qtbase
+# Maintainer: Timo Jyrinki <tjyrinki@suse.de>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use x11utils 'turn_off_gnome_screensaver';
+use version_utils 'is_sle';
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+
+sub run {
+    my $self = shift;
+    turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome');
+
+    # Install required packages
+    # designer-qt5 is in package libqt5-qttools, yast release notes in yast2-installation
+    # SDK needs to be added for 12-SP3
+    if (is_sle('<12-SP4')) {
+        select_console 'root-console';
+        cleanup_registration;
+        register_product;
+        add_suseconnect_product(get_addon_fullname('sdk'));
+    }
+    select_console 'x11';
```

Comment:
This might need an `ensure_unlocked_desktop`.

Reply:
Added.